### PR TITLE
WEB-3133: Stopping students to create unnecessary Labster accounts

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** 
Problem: CS has recently started receiving a lot of questions about enrollment from universities that are supposed to access the simulations on their own learning management systems.

Solution:
Adding the text below on the account creation page:

> Have you checked your course module on your school's learning management system?
>
> Labster simulations might have been already integrated into your course module. In this case, you do NOT need to create a Labster account. We highly recommend you to log in to your course module and check if you already have access to the simulations before creating an account.
>
> If you are not sure, please refer to your instructor.

screenshot from 2018-10-11 13-35-44

**JIRA:** https://liv-it.atlassian.net/browse/WEB-3133

**Sandboxes:** https://edx-eucalyptus.labster.com/

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer
- [ ] tag reviewer

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: lms/djangoapps/student_account/views.py [hunk 1]
```diff
@@ -104,11 +104,21 @@ def login_and_registration_form(request, initial_mode="login"):
...
+        "your course module and check if you already have access to the simulations before creating an account.<br /><br />"
+        "If you are not sure, please refer to your instructor."
+    )
+
     # Suggest student to create an account to the appropriate region server based on IP address of user.
     login_ip_address_warning_msg = register_ip_address_warning_msg = None
     if settings.LABSTER_FEATURES.get('ENABLE_REGION_IPADDR_WARNING'):
         regions = configuration_helpers.get_value('REGIONS', settings.REGIONS)
         current_region = request.session.get('country_code')
+        # current_region = "UK"
```
Comment (polesye) [inline lms/djangoapps/student_account/views.py]: remove?

File: lms/templates/student_account/register.underscore [hunk 2]
```diff
@@ -2,6 +2,10 @@
     <div class="status submission-warning" aria-live="polite">
         <p class="message-copy"><%= context.ipAddressWarningMsg %></p>
     </div>
+<% } else if (context.accountCreationWarningMsg) { %>
+    <div class="status submission-warning" aria-live="polite">
+        <p class="message-copy"><%= context.accountCreationWarningMsg %></p>
+    </div>
```
Comment (polesye) [inline lms/templates/student_account/register.underscore:4]: @lukyfebrianto please avoid duplications (html is the same, rename property to something general and use it)

File: lms/static/js/student_account/views/AccessView.js [hunk 3]
```diff
@@ -163,7 +163,8 @@
                         model: model,
                         thirdPartyAuth: this.thirdPartyAuth,
                         platformName: this.platformName,
-                        ipAddressWarningMsg: this.registerIpAddressWarningMsg // Added by Labster
+                        ipAddressWarningMsg: this.registerIpAddressWarningMsg, // Added by Labster
+                        accountCreationWarningMsg: this.accountCreationWarningMsg // Added by Labster
```
Comment (polesye) [inline lms/static/js/student_account/views/AccessView.js]: `// Added by Labster` was added here, but the line above not. What's the case? :)

**Comment by polesye** [comment]
should be tested before merging

**Comment by TamoshaytisV** [comment]
@lukyfebrianto @polesye PR was closed by accident
Was pressing "Cancel comment" button. Please reopen

File: lms/templates/student_account/register.underscore (hunk 2 above)
Comment (lukyfebrianto) [reply lms/templates/student_account/register.underscore:4]: renamed

File: lms/static/js/student_account/views/AccessView.js (hunk 3 above)
Comment (lukyfebrianto) [reply lms/static/js/student_account/views/AccessView.js]: added

File: lms/djangoapps/student_account/views.py (hunk 1 above)
Comment (lukyfebrianto) [reply lms/djangoapps/student_account/views.py]: good catch!
removed

**Review by TamoshaytisV — APPROVED** [review]

**Comment by polesye** [comment]
@lukyfebrianto is thje any documentation that explains the flow? Should it be updated?

**Review by polesye — APPROVED** [review]

**Comment by lukyfebrianto** [comment]
@polesye I think we don't need to update any documentation, we only add a new message and rename the property of warning message to something general.

**Pull request merged**